src/plugins/expressions/common/expression_functions/expression_function.ts

```diff
@@ -90,9 +92,11 @@ export class ExpressionFunction implements PersistableState<ExpressionAstFunctio
       inject,
       extract,
       migrations,
+      namespace,
     } = functionDefinition;
 
     this.name = name;
+    this.namespace = namespace;
```

Comment:
as described in the issue this pr is closing:

memory overhead, due to keeping some functions in multiple registries
maintenance overhead: if you are using fork you will need to add your own alert types, report types etc to get functionality which you would otherwise get out of the box.

yes, we considered fork implementation, its what we had until now, but there were problems with it and thus we decided to step away from it.